website/content/docs/platform/k8s/helm/examples/ha-tls.mdx

```diff
@@ -0,0 +1,102 @@
+---
+layout: 'docs'
+page_title: 'HA Cluster with TLS'
+sidebar_current: 'docs-platform-k8s-examples-ha-tls'
+description: |-
+  Describes how to set up a Raft HA Vault cluster with TLS certificate
+---
+
+# Raft HA server with TLS
+
+The overview for [Integrated Storage and
+TLS](/docs/concepts/integrated-storage#integrated-storage-and-tls) covers the
+various options for mitigating TLS verification warnings and bootstrapping your
+Raft cluster.
+
+The examples below demonstrate two specific solutions. Both solutions ensure
+that the common name (CN) used for the `leader_api_addr` in the Raft stanza
+matches the name(s) listed in the TLS certificate.
+
+## Before you start
+
+1. Follow the steps from the example [HA Vault Cluster with Integrated
+Storage](/docs/platform/k8s/helm/examples/ha-with-raft) to build the cluster.
+
+2. Follow the examples and instructions in [Standalone Server with
+TLS](/docs/platform/k8s/helm/examples/standalone-tls) to create a TLS
+certificate.
+
+## Bootstrap your Raft cluster
+
+Without proper configuration, you will see the following warning before cluster
+initialization:
+```shell
+core: join attempt failed: error="error during raft bootstrap init call: Put "https://vault-${N}.${SERVICE}:8200/v1/sys/storage/raft/bootstrap/challenge": x509: certificate is valid for ${SERVICE}, ${SERVICE}.${NAMESPACE}, ${SERVICE}.${NAMESPACE}.svc, ${SERVICE}.${NAMESPACE}.svc.cluster.local, not vault-${N}.${SERVICE}"
+```
```

Comment:
I'm not sure having this as an independent section is useful. What do you think of dropping the heading and making it the second paragraph in the introduction? That way you're outlining the situation in the first paragraph, showing the symptom in the second paragraph, then introducing your example solutions.